source/codegen/common_helpers.py

```diff
@@ -141,13 +141,13 @@ def pascal_to_camel(pascal_string):
 
 
 def ensure_pascal_case(pascal_or_camel_string):
```

Comment:
(Sorry) Just an indentation change. This method was out-of-sync with the rest of the file and was causing problems.